bugbot/rules/assignee_no_login.py

```diff
@@ -168,6 +174,35 @@ def get_bz_params(self, date):
 
         return params
 
+    def send_consolidated_email(self, date):
+        """Send a single email to each assignee with the list of unassigned bugs due to inactivity."""
+        for assignee, bugs in self.bugs_to_unassign.items():
+            self.add(assignee, bugs)
+
+        self.send_mails("[bugbot] Unassigned bugs due to inactivity")
+
+    def send_email(self, date="today"):
+        super().send_email(date)
```

Comment:
You do not need to overwrite the `send_email()` method when using the `Nag` class. You could use the `add()` method in `handle_inactive_assignees()`, and it will take care of preparing the data of the rest. Based on the escalation configuration, it will email the person and CC the manager of that person.

If multiple individuals are working under the same manager, the bugs will be grouped in the same email, and the manager will be in the CC.

Reply:
Done in 73bf20b2e9b433dd2f89f43faef301d8bd91f968!